.nais/nais-dev.yaml

```diff
@@ -40,3 +40,9 @@ spec:
       enabled: true
   secureLogs:
     enabled: true
+  env:
+    - name: PDL_SCOPE
+      value: api://dev-fss.pdl.pdl-api/.default
+    - name: PDL_URL
+      value: https://pdl-api.dev-fss-pub.nais.io
```

Comment:
Hvis vi skal kunne nå denne må den defineres som en tillatt ekstern host i outbound policy her i nais-yaml, ref det her: https://nav-it.slack.com/archives/C02CGB7RLKE/p1681218404378009 :)